x-pack/plugins/monitoring/public/views/base_controller.js

```diff
@@ -73,13 +73,35 @@ export class MonitoringViewBaseController {
   }) {
     const titleService = $injector.get('title');
     const $executor = $injector.get('$executor');
+    const $window = $injector.get('$window');
 
     titleService($scope.cluster, title);
 
     $scope.pageData = this.data = { ...defaultData };
     this._isDataInitialized = false;
     this.reactNodeId = reactNodeId;
 
+    const initialHash = $window.location.hash;
+    let lastHash = initialHash;
+    let deferTimer;
```

Comment:
Is this doing anything? I see it being used in `clearTimeout(deferTimer)` but I never see it set?

Reply:
Good catch! I actually meant to assign the `setTimeout` to it. Will fix it